srcpkgs/gtk4/template

```diff
@@ -31,9 +31,9 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://www.gtk.org/"
 #changelog="https://gitlab.gnome.org/GNOME/gtk/-/raw/main/NEWS"
-changelog="https://gitlab.gnome.org/GNOME/gtk/-/raw/gtk-4-14/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/gtk/-/raw/4.16.0/NEWS"
```

Comment:
This should be ` changelog="https://gitlab.gnome.org/GNOME/gtk/-/raw/gtk-4-16/NEWS"`